app/services/mailSender.js

```diff
@@ -0,0 +1,19 @@
+const config = require('../../config').common.email,
+  transport = require('nodemailer').createTransport(config),
+  { registrationHtml } = require('../helper/htmlMessages'),
+  logger = require('../logger');
+
+const sendConfirmationEmail = user =>
```

Comment:
I'd promisify this, but as u wish

Reply:
Hm, when is acceptable to promisify something? I remember that you said it wasn't something too acceptable

Reply:
this has callbacks, it's asynchronous. What I said was that it is wrong to promisify synchronous things